test/compare.test.ts

```diff
@@ -0,0 +1,136 @@
+import { describe, expect, it } from 'vitest'
+
+import type { SortingNode } from '../typings'
+
+import { compare } from '../utils/compare'
+
+describe('compare', () => {
+  describe('alphabetical', () => {
+    let compareOptions = {
+      specialCharacters: 'keep',
+      type: 'alphabetical',
+      ignoreCase: false,
+      locales: 'en-US',
+      order: 'asc',
+    } as const
+
+    it('sorts by order asc', () => {
+      expect(
+        compare(createTestNode({ name: 'b' }), createTestNode({ name: 'a' }), {
+          ...compareOptions,
+        }),
```

Comment:
```suggestion
        compare(
          createTestNode({ name: 'b' }),
          createTestNode({ name: 'a' }),
          compareOptions,
        ),
```

Reply:
@azat-io Oops! I trusted ESLint and prettier to fix that for me 😅